src/bin_average_comp.F90

```diff
@@ -80,7 +80,7 @@ program bin_average_comp
 
   output_type = OUTPUT_TYPE_SINGLE
   record_removals = .false.
-  record_optical = .true.
+  record_optical = .false.
```

Comment:
I will have to dig into this to see if there's unintended consequences of this change. The issue is that assuming true always causes segmentation faults now that we have allocatable arrays instead of scalar values if you average populations that didn't originally have optical properties.

I think the solution is to check if optical properties were a part of the original simulation.

Reply:
Found the underlying issue and changed this back to true. https://github.com/compdyn/partmc/pull/168/commits/ed2dd5c9c92b5aca4db7518f42370d0e994899ca